contrib/gitlab.tpl

```diff
@@ -29,6 +29,10 @@
   {{- range . }}
   {{- $target := .Target }}
     {{- $image := $target | regexFind "[^\\s]+" }}
+    {{- $os := "Unknown" }}
+    {{- if contains "(" $target -}}
+      {{- $os = regexReplaceAll ".+\\((.+)\\)" $target "${1}" }}
```

Comment:
We try to avoid using `regex` if possible.
Can we use [split](http://masterminds.github.io/sprig/string_slice.html) + [trimSuffix](http://masterminds.github.io/sprig/strings.html) functions?